src/CheckRightsService.Broker/Consumers/AccessCollectionValidatorConsumer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using LT.DigitalOffice.CheckRightsService.Data.Interfaces;
+using LT.DigitalOffice.Kernel.AccessValidatorEngine.Requests;
+using LT.DigitalOffice.Kernel.Broker;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.CheckRightsService.Broker.Consumers
+{
+    public class AccessCollectionValidatorConsumer : IConsumer<IAccessValidatorCheckRightsCollectionServiceRequest>
+    {
+        private readonly ICheckRightsRepository repository;
```

Comment:
Now we use '_' before private field, like _repository

Reply:
fixed